autogen/docs/behaviorspace.md.mustache

```diff
@@ -360,6 +360,23 @@ metrics are in orange, and the measurement metric data is in purple.
 
 ![[bs-spreadsheet.png]]
 
+If the **Statistics output** is enabled, data in either the specified **Table output** or
+**Spreadsheet output** is used to calculate the mean and standard deviation of each numeric
+metric across repetitions for each step. These calculations are done at the end of the experiment.
+The statistics are then saved in the specified file. It is important to note that metrics
+that may produce non-numeric measurements like strings are not included in the statistics. If
+the metrics produce lists, the statistics are calculated across elements with the same index for each list.
+Statistics are not calculated for metrics that produce lists that contain non-numeric elements. Also, the type of
+the measurement must remain constant. For example, statistics are not calculated if a metric
+produces measurements that can be both lists and numbers. Furthermore, the standard deviation may be "N/A",
+which means that there were two or fewer collected measurements for that step and parameter combination,
+producing a result that is not well-defined.
```

Comment:
Maybe add a note saying explicitly that the experiment must use a **Repetitions** setting of 3 or more here in its settings (or whatever other settings are needed) to get the standard deviation to populate.